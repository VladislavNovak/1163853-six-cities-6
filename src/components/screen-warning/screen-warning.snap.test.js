import React from 'react';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';

import {ScreenWarning} from '..';

test(`Should ScreenWarning render correctly`, () => {
  const history = createMemoryHistory();
  const warning = `...LOADING...`;

  const {container} = render(
      <Router history={history}>
        <ScreenWarning warning={warning}/>
      </Router>);
  expect(container).toMatchSnapshot();
});
