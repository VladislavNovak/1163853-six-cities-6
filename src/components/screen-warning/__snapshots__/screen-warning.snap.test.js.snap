// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ScreenWarning render correctly 1`] = `
<div>
  <div
    class="sc-bdfBwQ dDvbHG"
  >
    <div
      class="sc-gsTCUz gFObRd"
    >
      <div
        class="sc-dlfnbm fRMjel"
      >
        <div
          class="sc-hKgILt eRjvFP"
        />
        <h1
          class="sc-eCssSg kxcsyR"
        >
          404
        </h1>
      </div>
      <h2
        class="sc-jSgupP kDwJh"
      >
        ...LOADING...
      </h2>
      <p
        class="sc-gKsewC eCBzea"
      >
        The page you are looking for might have been removed had its name changed or is temporarily unavailable.
      </p>
      <a
        class="sc-iBPRYJ flrGhZ"
        href="/"
      >
        Вернуться на главную
      </a>
    </div>
  </div>
</div>
`;
